### ISSUE TITLE
Failed to call service vacuum/start. 502 : send command.

### ISSUE THREAD

**Issue by Aham330** [py-smart-gardena/hass-gardena-smart-system#178]
I took the power off the Mower for a few minutes. After putting the power back on the integration fails to work. 

I get a message Failed to call service vacuum/start. 502 : send command.

I have uninstalled the integration, restarted HA and then reinstalled (with the same API) but it still issue still remains.

Could you have a look.

**Comment by danroeg** [comment]
@Aham330 have you recently done a firmware upgrade? I have 502 bad gateway problems and mower not starting on START_SECONDS_TO_OVERRIDE since the last update

**Comment by Aham330** [comment]
Hi @danroeg thanks for you reply.

unless the mower auto updated itself after having switched off the mower I dont think I updated. Nor do I think it did that cause I don’t have auto updates on. 

The mower app works fine and starting my vacuum robot works fine too.

**Comment by gijs007** [comment]
I'm having similar issues here. The mower works fine in the app, but home assistant can no longer start it.

If I recall correctly there was an update for the mower a couple of weeks ago, which had to manually be installed in the app and required a reboot.
Even after this update everything was fine, until a couple of days ago.

**Comment by danroeg** [comment]
yes sounds familiar. please also write to husqvarna dev team to fix the api

**Comment by grm** [comment]
Hello,

My mower felt off and is not connected for a few weeks now.

I'll try to find some time this week end to install it and reproduce
the issue.

**Comment by Aham330** [comment]
Hi @grm 

Any luck with the reproduction?

greets 

Aham

**Comment by danroeg** [comment]
Hello, any updates here?

**Comment by j3mu5** [comment]
This is a problem for me as well. I get the error `vacuum/start. 502 : connection lost`

**Comment by Aham330** [comment]
OK. This stuff never ceases to amaze me. 

After just pressing play in Home Assistant today to get my Mower going and mowing, it, contrary to all expectations, left the garage and started. Funny thing being that I did not change anything on either side of the installation. So not on the side of the Mower and not on the side of HA.

What I did do following the suggestion of @danroeg is write the following to Husqvarna: 

<img width="872" alt="Scherm­afbeelding 2023-07-25 om 11 52 13" src="https://github.com/py-smart-gardena/hass-gardena-smart-system/assets/73581209/0a55cb3f-697c-4c6c-bcf0-12430f65abc2">

Should Husqvarna have been involved in fixing this issue I would like express my thanks and at same time ask to comment either here or reply to the email I sent. So as to establish a two communication with which we could possibly float idea's for the greater community and your costumers.

Aham.

**Comment by danroeg** [comment]
@Aham330  thanks for the update - I’ve tested and indeed I can once again manually start the mower, which is a great relief. Thank you for chasing Husqvarna and good luck with the pets.

**Comment by grm** [comment]
Closing — this was a v1.x issue. The integration has been completely rewritten in v2.x with a new architecture (no more py-smart-gardena dependency, native websocket handling, proper token refresh, LawnMowerEntity support). Please upgrade to v2.x via HACS. If you still experience issues on v2.x, please open a new issue.
